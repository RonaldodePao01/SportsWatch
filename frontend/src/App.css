.App {
  text-align: center;
  font-family: "roboto";
}

body {
  background-color: #778da9;
}

.login-card {
  height: 70vh;
  width: 40%;
  background-color: #415a77;
  margin: 5em 0 0 5em;
  padding: 2em 0;
  border-radius: 20px;
}

.welcome {
  width: 50%;
  background-color: black;
  margin: 0 auto;
  padding: 1em 0.5em;
  border-radius: 20px;
  color: white;
}

.login-card input {
  border-radius: 5px;
  border: 0;
  width: 50%;
  padding: 0.7em;
  margin: 0.4em;
}

.login-card button {
  background-color: black;
  color: white;
  border-radius: 5px;
  margin-top: 0.2em;
  padding: 0.7em 2em;
  border: none;
}

.flex {
  display: flex;
}

.lrg-heading {
  align-self: center;
  line-height: 0.5;
  max-width: 45%;
  margin: 0 auto;
  font-size: 50px;
}

.logout-btn {
  color: black;
  background-color: white;
  border-radius: 5px;
  border: none;
  padding: 0.1em 0.5em;
  margin-right: 1em;
  margin-bottom: 0.4em;
  font-size: 13px;
}

.welcome-back {
  font-size: 36px;
  margin-bottom: 0.3em;
  background-color: #0d1b2a;
  color: white;
  margin: 0 auto;
  /* margin-top: 1em !important; */
  margin-bottom: 0.8em !important;
  padding: 0.4em;
  width: 20%;
  border-radius: 20px;
  text-align: center;
}

.nav {
  display: flex;
  background-color: black;
  color: white;
  justify-content: space-between;
  position: fixed;
  width: 100%;
  top: 0;
}

.nav h5 {
  padding: 0;
}

.name {
  margin: 0.5em 0 0.5em 1em;
  font-size: 20px;
}

h5 {
  align-self: center;
  margin: 0.4em 1em 0 0;
  padding-top: 0.5em;
  color: white;
}

.match-card {
  border-radius: 10px;
  width: 45%;
  list-style: none;
  margin: 0 auto;
  margin-bottom: 2em !important;
  padding: 1em;
  background-color: #e0e1dd;
}

.match-card h4,
h2,
h3 {
  margin: 0.2em;
  max-width: 80%;
  margin: 0 auto;
  margin-top: 0.2em !important;
}

.match-card button {
  background-color: #415a77;
  color: white;
  border: none;
  border-radius: 5px;
  padding: 0.3em 0.5em;
}

.date {
  background-color: #0d1b2a;
  color: white;
  width: 12%;
  margin: 0.05em;
  border-radius: 5px;
  font-weight: 400;
}

.space {
  display: flex;
  text-align: center;
  justify-content: center;
  width: 100%;
  gap: 1em;
  margin: 0.5em 0;
}

ul {
  padding: 0;
}

.favourites {
  height: 100%;
  padding: 0;
  align-self: center;
  justify-content: center;
  padding: 0.2em 0.4em;
  border-radius: 5px;
  border: none;
  margin-top: 3em;
  margin-bottom: 1em;
}

.spacing {
  margin-top: 3em;
}

.fav-heading {
  font-size: 36px;
  margin-bottom: 0.3em;
  background-color: #0d1b2a;
  color: white;
  margin: 0 auto;
  margin-bottom: 0.8em !important;
  padding: 0.4em;
  width: 20%;
  border-radius: 20px;
  text-align: center;
}

button:hover {
  cursor: pointer;
}

.long-input {
  width: 100%;
  height: 2e;
}

.admin {
  padding: 0;
  margin: 0;
  margin-top: 0.5em;
}

.admin-spacing {
  margin-top: 5em;
}

.admin-logout {
  margin-top: 1em;
}

.add-event {
  background-color: #e0e1dd;
  width: 50%;
  margin: 0 auto;
  text-align: center;
  padding: 0.2em 1em 0.5em 1em;
  border-radius: 10px;
}

.pad {
  margin: 0.2em;
}

@media screen and (max-width: 920px) {
  .lrg-heading {
    display: none !important;
  }
  .login-card {
    width: 100%;
    margin: 4em 2em !important;
  }
  .welcome {
    font-size: 10px;
  }
  h2 {
    font-size: 15px;
  }
  .welcome-back {
    font-size: 24px;
    width: 60%;
    border-radius: 10px;
  }
  .fav-heading {
    font-size: 24px;
    width: 60%;
    border-radius: 10px;
  }
  .add-event {
    width: 80%;
  }
  h1 {
    font-size: 20px;
  }
  .match-card {
    width: 80%;
    font-size: 0.8em;
  }
  .match-card h3,
  h2,
  h4 {
    padding: 0.2em;
  }
  .space {
    display: flex;
    text-align: center;
    justify-content: center;
    width: 100%;
    gap: 1em;
  }

  .date {
    width: 20%;
    background-color: #0d1b2a;
  }
}
