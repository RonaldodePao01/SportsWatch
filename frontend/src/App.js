import "./App.css";
// importing the getMatches component
import GetMatches from "./components/GetMatches";

function App() {
  return (
    <div className="App">
      {/* making an instance of the getMatches child component in the main app component */}
      <GetMatches />
    </div>
  );
}

export default App;
