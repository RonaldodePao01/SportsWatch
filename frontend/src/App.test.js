import App from "./App";
import React from "react";
import renderer from "react-test-renderer";

it("snapshot test of app component", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
