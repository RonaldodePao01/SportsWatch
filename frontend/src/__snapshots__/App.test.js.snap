// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test of app component 1`] = `
<div
  className="App"
>
  <div
    className="flex"
  >
    <div
      className="login-card"
    >
      <div
        className="welcome"
      >
        <h1>
          Welcome to SportsWatch
        </h1>
      </div>
      <h2>
        Log in to access your account
      </h2>
      <input
        placeholder="Username"
      />
      <input
        placeholder="Password"
      />
      <div>
        <button
          onClick={[Function]}
        >
          Log in
        </button>
      </div>
      <div>
        <h2>
          Sign up
        </h2>
        <input
          placeholder="Username"
        />
        <input
          placeholder="Password"
        />
        <div>
          <button
            onClick={[Function]}
          >
            Sign up
          </button>
        </div>
      </div>
    </div>
    <div
      className="lrg-heading"
    >
      <h1>
        Get the 
      </h1>
      <h1>
        latest
      </h1>
      <h1>
        sporting
      </h1>
      <h1>
        fixtures
      </h1>
    </div>
  </div>
</div>
`;
